libraries/AP_DDS/README.md

```diff
@@ -274,14 +278,44 @@ or
 std_srvs.srv.Trigger_Response(success=True, message='Vehicle is Armable')
 ```
 
-Call the takeoff service:
+
+Clear the Rally Points list (the provided Point will be ignored):
+
+```bash
+ros2 service call /ap/rally_set ardupilot_msgs/srv/RallySet "{clear: true}"
+requester: making request: ardupilot_msgs.srv.RallySet_Request(rally=ardupilot_msgs.msg.Rally(point=geographic_msgs.msg.GeoPoint(latitude=0.0, longitude=0.0, altitude=0.0), altitude_frame=0), clear=True)
+
+response:
+ardupilot_msgs.srv.RallySet_Response(success=True, size=0)
+```
+
+Append a Rally Point:
+
+Rally points units of measure are:
+* degrees for the latitude and longitude (up to 7 decimal digits resolution);
+* meters for the altitude (with no decimals).
+
+The `altitude_frame` specifies the datum for the altitude, among Home, Absolute, Terrain or EKF Origin
+(use the enumerator specified in the message definition).
 
 ```bash
-$ ros2 service call /ap/experimental/takeoff ardupilot_msgs/srv/Takeoff "{alt: 10.5}"
-requester: making request: ardupilot_msgs.srv.Takeoff_Request(alt=10.5)
+ros2 service call /ap/rally_set ardupilot_msgs/srv/RallySet "{rally: {altitude_frame: 1,  point: {latitude: -35.12345, longitude: 151.12345, altitude: 400.5}}}"
+
+requester: making request: ardupilot_msgs.srv.RallySet_Request(rally=ardupilot_msgs.msg.Rally(point=geographic_msgs.msg.GeoPoint(latitude=-35.12345, longitude=151.12345, altitude=400.5), altitude_frame=1), clear=False)
```

Comment:
At line 296 it says altitude has no decimals. Also doesn't this return a size now?

Reply:
I fixed that